### PR TITLE
Adding 2nd example

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/94

### CODE REVIEW

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Revoke-SPOUserSession.md [hunk 1]
```diff
@@ -46,14 +46,22 @@ The cmdlet will be available in the future, but it isn't ready for use in your o
...
 ## EXAMPLES
 
-### ----------------------EXAMPLE-----------------------
+### EXAMPLE 1
 
 ```powershell
 Revoke-SPOUserSession -User user1@contoso.com
 ```
 
 This example signs out user1 in the contoso tenancy from all devices.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Revoke-SPOUserSession.md:55]:
```suggestion
This example signs out user1 in the Contoso tenancy from all devices.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Revoke-SPOUserSession.md [hunk 2]
```diff
@@ -46,14 +46,22 @@ The cmdlet will be available in the future, but it isn't ready for use in your o
...
 
+### EXAMPLE 2
+
+```powershell
+Revoke-SPOUserSession -User user1@contoso.com -Confirm:$false
+```
+
+This example signs out user1 in the contoso tenancy from all devices without prompting for confirmation.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Revoke-SPOUserSession.md:63]:
```suggestion
This example signs out user1 in the Contoso tenancy from all devices without prompting for confirmation.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks.

**Pull request merged**